ydb/core/protos/config.proto

```diff
@@ -290,6 +290,7 @@ message TBlobStorageConfig {
         optional NKikimrBlobStorage.EPDiskType PDiskType = 1;
         optional uint64 BurstThresholdNs = 2;
         optional float DiskTimeAvailableScale = 3;
+        optional uint32 MinHugeBlobSizeInBytes = 4;
```

Comment:
It is not actually "cost metrics". Either rename the structure (and retain textual compatibility if needed), either add a new one.